### ISSUE TITLE
Paypal is disabled for PROJECT_VERSION_MAJOR != '1'

### ISSUE THREAD

**Issue by torvista** [zencart/zencart#5957]
For ZC200, Paypal does not display due to this:
https://github.com/zencart/zencart/blob/3423fe39ff4dfeb7429c3b6ab29cf02a3e507bf5/includes/modules/payment/paypalwpp.php#L217

**Comment by scottcwilson** [comment]
Nice catch!

**Comment by drbyte** [comment]
I wonder why it displayed fine for 1.5.8? ... since `5.6` != `5.8` ... 🤔

**Comment by scottcwilson** [comment]
It stops at the first condition since AND is used.  It doesn't check PROJECT_VERSION_MINOR

**Comment by drbyte** [comment]
That doesn't explain why it DOES work on 1.x when "5.6" and "5.8" don't match...

**Comment by torvista** [comment]
I don't understand the doubt...
```
define('PROJECT_VERSION_MAJOR', '1');
define('PROJECT_VERSION_MINOR', '5.8a');
if (PROJECT_VERSION_MAJOR != '1' && substr(PROJECT_VERSION_MINOR, 0, 3) != '5.6') $this->enabled = false;
```
PROJECT_VERSION_MAJOR != '1' is false, so the check fails  and the module continues being enabled.
So the module would be enabled for all 1.x versions...what was the point of this check at all!

**Comment by drbyte** [comment]
🤦‍♂️

**Comment by drbyte** [comment]
> what was the point of this check at all

Many versions ago it was intended to ensure that a store didn't use a mismatched PayPal module on a version of Zen Cart it wasn't written for. At one point there were people copying newer versions of the module onto older carts without upgrading the cart and the module would break because it had dependencies on other parts of Zen Cart that needed upgrading too (such as functions files, etc).

At one point this "check" did work. I guess somewhere along the line the logic got messed up when updating version numbers etc.

I think we can safely remove the check altogether.

**Comment by drbyte** [comment]
Closed by #5963
